docs/docs/api/turn-by-turn/api-reference.md

```diff
@@ -113,6 +115,7 @@ These options are available for `auto`, `bus`, and `truck` costing methods.
 | `country_crossing_cost` | A cost applied when encountering an international border. This cost is added to the estimated and elapsed times. The default cost is 600 seconds. |
 | `country_crossing_penalty` | A penalty applied for a country crossing. This penalty can be used to create paths that avoid spanning country boundaries. The default penalty is 0. |
 | `shortest` | Changes the metric to quasi-shortest, i.e. purely distance-based costing. Note, this will disable all other costings & penalties. Also note, `shortest` will not disable hierarchy pruning, leading to potentially sub-optimal routes for some costing models. The default is `false`. |
+| `disable_hierarchy_pruning` | Disable hierarchy pruning to calculate the actual shortest distance. **Note:** This could be quite a performance drainer so there is a upper limit of distance. If the upper limit is exceeded, this option will always be `false`. The default is `false`. |
```

Comment:
also here, not only shortest, but rather actual optimal route. even though @kevinkreiser might not agree, maybe there's other stuff which prevents a real optimal path (note, with hierarchies it's in >95% also optimal, but there's lots of edge cases)

Reply:
Ack.